TYPO3.Neos/Resources/Public/JavaScript/Content/Application.js

```diff
@@ -406,6 +406,12 @@ function(
 						$('title').html($htmlDom.filter('title').html());
 						$('link[rel="neos-site"]').attr('href', $htmlDom.filter('link[rel="neos-site"]').attr('href'));
 
+						var $pageStylesheets = $('link[rel="stylesheet"]:not(.neos-ui-asset)');
+						$pageStylesheets.remove();
+
+						var $loadedStylesheets = $htmlDom.filter('link[rel="stylesheet"]:not(.neos-ui-asset)');
+						$('head').append($loadedStylesheets);
```

Comment:
removing the stylesheet and adding it again can easily have side effects in various browsers (e.g. cause unnecessary reflow + paint, cache flushing and reloading of assets).. it would be more robust just to check for it's existence and insert it if it doesn't exist..